shimmingtoolbox/unwrap/unwrap_phase.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+""" Wrapper to different unwrapping algorithms
+
+"""
+
+
+import numpy as np
+from shimmingtoolbox.unwrap.prelude import prelude
+
+
+def unwrap_phase(complex_array, affine, unwrap_function="prelude", mask=np.array([-1])):
+    """ Calls different unwrapping algorithms according to the specified `unwrap_function` parameter
+
+    Args:
+        complex_array (numpy.ndarray): 2D,3D,4D or 5D complex values numpy array to perform phase unwrapping
```

Comment:
i suggest to input mag and phase as 2 separate arguments. we could make mag optional